pkg/controller/volume/expand/cache/volume_resize_map.go

```diff
@@ -160,6 +163,21 @@ func (resizeMap *volumeResizeMap) MarkAsResized(pvcr *PVCWithResizeRequest, newS
 	return nil
 }
 
+// MarkForFSResize marks pvc with condition that indicates a fs resize is pending
+func (resizeMap *volumeResizeMap) MarkForFSResize(pvcr *PVCWithResizeRequest) error {
+	pvcCondition := v1.PersistentVolumeClaimCondition{
+		Type:               v1.PersistentVolumeClaimFileSystemResizePending,
+		Status:             v1.ConditionTrue,
+		LastTransitionTime: metav1.Now(),
+		Message:            "Waiting for user to (re-)start a pod to finish file system resize of volume on node.",
+	}
+	conditions := []v1.PersistentVolumeClaimCondition{pvcCondition}
+	newPVC := pvcr.PVC.DeepCopy()
+	newPVC = util.MergeResizeConditionOnPVC(newPVC, conditions)
```

Comment:
take a look at what other condition updaters do... if the PersistentVolumeClaimFileSystemResizePending condition already exists with the status ConditionTrue, I wouldn't expect LastTransitionTime to be updated on the object. as written, MergeResizeConditionOnPVC doesn't consider whether the condition has actually updated, and always results in a write

Reply:
I am not sure - that is true.

If `PersistentVolumeClaimFileSystemResizePending` already exists on the PVC and user has requested another resize while resize was pending on the node, the code will try and resize the PVC again and set a new `PersistentVolumeClaimFileSystemResizePending` with a new `LastTransitionTime` and it would be incorrect to not update it..

Reply:
> If PersistentVolumeClaimFileSystemResizePending already exists on the PVC and user has requested another resize while resize was pending on the node, the code will try and resize the PVC again and set a new PersistentVolumeClaimFileSystemResizePending with a new LastTransitionTime and it would be incorrect to not update it..

I don't think that's accurate... in that scenario, the condition status has not changed, so the last transition time should not change

Reply:
fixed

Reply:
I have fixed this as per ^ discussion. But I am curious - why  does it matter? What happens if same condition (with same status) does end up changing `LastTransitionTime`?

Reply:
that tends to lead to what should be a no-op API write continuously generating changes/writes/watch events

Reply:
ack. can you please approve the PR?